pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -484,6 +484,7 @@ public void setupInput() throws Exception {
         } catch (Exception e) {
             log.info("Error occurred executing open for source: {}",
                     sourceSpec.getClassName(), e);
+            throw e;
```

Comment:
We shouldn't be logging the exception multiple times or it can be confusing. If we're re-throwing there should be no log here. 

To add context we can do like : 

```java
throw new IOException("Failed to open sink", e);
```

Reply:
Removed logging